src/shapes.ts

```diff
@@ -108,13 +113,11 @@ export interface WebOptions {
 }
 export interface OpenFinWindow extends InjectableContext {
     isIframe?: boolean;
-    parentFrameId?: number;
     _options: WindowOptions;
```

Comment:
we sure we dont need parentFramId in runtime?

Reply:
Looks like a core thing only